Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -800,9 +819,10 @@ protected class ClassWriter : IClassWriter {
         //TODO: Add deeper types
         return "Callable";
       }
-
       var cl = udt.ResolvedClass;
+
       return cl switch {
+        ArrayClassDecl => "list",
```

Comment:
Going by your Runtime, this should be an Array. Is there a reason not to make this a sequence?

Reply:
?